.devops/templates/tools.yml

```diff
@@ -3,6 +3,7 @@ steps:
   - task: NodeTool@0
     inputs:
       versionSpec: '12.x'
+      checkLatest: true
```

Comment:
**q:** why is this needed ?

Reply:
`eslint-plugin-jsdoc` for some reason has an `engines` config specifying the very latest version of node 12. The [node installer task](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/node-js?view=azure-devops) uses a cached node version by default rather than checking every time for the latest version (I'm not sure what the cache expiration interval is), so some of our build VMs were still on old versions. So adding this option forces them to update.